### ISSUE TITLE
`dvc`: prevent printing html path

### ISSUE THREAD

**Issue by sirily** [iterative/dvclive#177]
I'm using dvclive with pytorch lightning in dvc pipeline. While training there is a string with the path to metrics html, and it breaks progress bar and fills the console with useless messages. Is there a way to disable printing this line?
> Epoch 10:   8%|█▊                      | 103/1361 [00:11<02:13,  9.42it/s, loss=1, v_num=_run, train_positive_step=0.988]
> file:///path/index.html
> Epoch 10:  15%|███▋                    | 209/1361 [00:22<02:01,  9.51it/s, loss=1, v_num=_run, train_positive_step=0.985]
> file:///path/index.html
> Epoch 10:  22%|█████▍                  | 305/1361 [00:32<01:50,  9.53it/s, loss=1, v_num=_run, train_positive_step=0.991]
> file:///path/index.html
> Epoch 10:  29%|███████                 | 400/1361 [00:42<01:40,  9.53it/s, loss=1, v_num=_run, train_positive_step=0.984]
> file:///path/index.html
> Epoch 10:  37%|████████▉               | 505/1361 [00:53<01:29,  9.53it/s, loss=1, v_num=_run, train_positive_step=0.990]
> file:///path/index.html

Inside train stage logger is defined like this:
```
logger = DvcLiveLogger()
model = Trainer(hparams, model_path)
trainer = pl.Trainer(
                    gpus=1,
                    logger=logger,
                    max_epochs=hparams['num_epochs'],
                    log_every_n_steps=hparams['log_every_n_steps'],
                    callbacks=[checkpoint_callback, early_stopping],
                    stochastic_weight_avg=True)
```
in dvc.yaml stage looks like this:
```
cmd: python3 pipeline/train.py --task ${item.task} --inp_dir ${item.in} --out_dir ${item.out}
deps:
- data/processed/train_${item.task}_df.par
- data/processed/val_${item.task}_df.par
- models/${item.in}
- pipeline/train.py
- src/trainer.py
params:
- ${item.task}_train
outs:
- models/${item.out}
live:
  metrics/${item.out}_train:
    cache: false
    summary: true
    html: true
```
It seems like nothing bad happens in dvclive, maybe it prints because of summary path [here](https://github.com/iterative/dvc/blob/e5cf9b203df01f13e5568c745906d74bc131235c/dvc/repo/live.py#L25). But @shcheklein asked me to open an issue here.

dvc doctor:

> DVC version: 2.7.4 (pip)

> ---------------------------------
> Platform: Python 3.8.10 on Linux-5.4.0-72-generic-x86_64-with-glibc2.29
> Supports:
>         hdfs (pyarrow = 5.0.0),
>         http (aiohttp = 3.7.4.post0, aiohttp-retry = 2.4.5),
>         https (aiohttp = 3.7.4.post0, aiohttp-retry = 2.4.5),
>         s3 (s3fs = 2021.8.1, boto3 = 1.17.106)
> Cache types: hardlink, symlink
> Cache directory: ext4 on /dev/mapper/vg0-root
> Caches: local
> Remotes: s3
> Workspace directory: ext4 on /dev/mapper/vg0-root
> Repo: dvc (subdir), git

dvclive version: 0.4.1
